pusu.js.org

xliudaxia opened the pull request:
- [x] There is reasonable content on the page (see: [No Content](https://github.com/js-org/js.org/wiki/No-Content))
- [x] I have read and accepted the [Terms and Conditions](http://js.org/terms.html)

MattIPv4 reviewed (approved):
✅ Commit valid, CNAME present, site appears to have content \*

\* based off https://github.com/xliudaxia/pusu-ui/tree/gh-pages

indus commented:
I've already added your requested subdomain to JS.ORGs zonefile. But you have to fix the path issue to make it work.
I'll merge after I had a look at your page and approved it. Please keep this PR open.

xliudaxia commented:
> 



> I've already added your requested subdomain to JS.ORGs zonefile. But you have to fix the path issue to make it work. I'll merge after I had a look at your page and approved it. Please keep this PR open.

I have already fixed the path issue , and my site now working sucessfully under this domain,thanks,please approve my request !

indus commented:
Welcome https://pusu.js.org/
